examples/data-sources/hpegl_metal_available_resources/README.md

```diff
@@ -1,6 +1,7 @@
+# (C) Copyright 2020-2023 Hewlett Packard Enterprise Development LP
```

Comment:
I'm thinking this should be a comment instead. Otherwise the copyright shows up as a H1 size.
```suggestion
<!-- (C) Copyright 2020-2023 Hewlett Packard Enterprise Development LP -->
```